server/service/grpc/transaction_service.rs

```diff
@@ -1585,3 +1698,9 @@ impl QueryStreamTransmitter {
         Continue(())
     }
 }
+
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub enum QueryQueueOptions {
```

Comment:
I think this is just `QueueOptions`, and then you have Query/Analyze within that. Analyze isn't a query itself